### PR TITLE
Remove missing parameter pass from SSH rules

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Remove missing parameter pass from SSH rules. Require explicit configuration setting in files.

#### Rationale:

- Related to: https://github.com/ComplianceAsCode/content/pull/8127
- Related to: https://github.com/ComplianceAsCode/content/pull/7827

### CODE REVIEW

**Comment by jhrozek** [comment]
/retest
the OCP CI should now be fixed (if not, maybe rebasing on master to make sure the current OCP fixes are pulled in should help)

**Review by Mab879 — CHANGES_REQUESTED** [review]
This will be nice addition to the project. 

Please rebase this PR and I will review.

**Comment by ggbecker** [comment]
> This will be nice addition to the project.
> 
> Please rebase this PR and I will review.

Rebased.

**Review by Mab879 — APPROVED** [review]

File: shared/templates/sshd_lineinfile/template.py [hunk 1]
```diff
@@ -3,4 +3,10 @@
 
 def preprocess(data, lang):
     data["missing_parameter_pass"] = parse_template_boolean_value(data, parameter="missing_parameter_pass", default_value=False)
+
+    is_default_value = parse_template_boolean_value(data, parameter="is_default_value", default_value=False)
```
Comment (Mab879) [inline shared/templates/sshd_lineinfile/template.py:7]: Line is too long, 107 > 99. Might be worth it to break it up.

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Review by Mab879 — APPROVED** [review]

**Pull request merged**